src/modules/colorPicker/ColorPicker/dllmain.cpp

```diff
@@ -0,0 +1,188 @@
+// dllmain.cpp : Defines the entry point for the DLL application.
+#include "pch.h"
+#include <common/common.h>
+#include <interface/powertoy_module_interface.h>
+#include "trace.h"
+#include <common\settings_objects.h>
+#include <common\os-detection\os-detect.h>
+
+extern "C" IMAGE_DOS_HEADER __ImageBase;
+
+BOOL APIENTRY DllMain( HMODULE hModule,
+                       DWORD  ul_reason_for_call,
+                       LPVOID lpReserved
+                     )
+{
+    switch (ul_reason_for_call)
+    {
+    case DLL_PROCESS_ATTACH:
+        Trace::RegisterProvider();
+    case DLL_THREAD_ATTACH:
+    case DLL_THREAD_DETACH:
+    case DLL_PROCESS_DETACH:
+        Trace::UnregisterProvider();
+        break;
+    }
+    return TRUE;
+}
+
+struct ModuleSettings{
+} g_settings;
+
+class ColorPicker : public PowertoyModuleIface
+{
+private:
+    bool m_enabled = false;
+
+    std::wstring app_name;
+
+    HANDLE m_hProcess;
+
+     // Time to wait for process to close after sending WM_CLOSE signal
+    static const int MAX_WAIT_MILLISEC = 10000;
+public:
+    ColorPicker()
+    {
+        app_name = L"ColorPicker";
+    }
+
+    ~ColorPicker()
+    {
+        if (m_enabled)
+        {
+        }
+        m_enabled = false;
+    }
+
+    // Destroy the powertoy and free memory
+    virtual void destroy() override
+    {
+        delete this;
+    }
+
+    // Return the display name of the powertoy, this will be cached by the runner
+    virtual const wchar_t* get_name() override
+    {
+        return app_name.c_str();
+    }
+
+    virtual const wchar_t** get_events() override
+    {
+        static const wchar_t* events[] = { nullptr };
+
+        return events;
+    }
+
+    virtual bool get_config(wchar_t* buffer, int* buffer_size) override
+    {
+        HINSTANCE hinstance = reinterpret_cast<HINSTANCE>(&__ImageBase);
+
+        // Create a Settings object.
+        PowerToysSettings::Settings settings(hinstance, get_name());
+        settings.set_description(L"Color picker");
+        //  settings.set_description(GET_RESOURCE_STRING(IDS_LAUNCHER_SETTINGS_DESC));
+
+        settings.set_overview_link(L"https://aka.ms/PowerToysOverview_ColorPicker");
```

Comment:
@crutkas will you be creating this link?

Reply:
link is created